### DISCUSSION TITLE
How do I customize the character encoding?

### DISCUSSION THREAD

**Discussion by nihs903222** [Eugeny/tabby#7696] [Q&A]
Hi, I've been looking into the issues.
There seems to be no way to modify the character encoding.
Because of my shell scripts and  docs are saved as Big5.
UTF-8 doesn't work.
I wonder if there is anyway to change it from UTF-8 to Big5.
It's too bad for such a good terminal without this function.

**Comment by UNiXMIT** [comment]
I love this software but i'm torn about moving from Termius now. Might have to keep both for now.
